test/blink/2-keyframes-with-offsets.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<script src="testharness/testharness.js"></script>
-<script src="testharness/testharnessreport.js"></script>
+<script src="../web-platform-tests/resources/testharness.js"></script>
```

Comment:
I think these three script imports can now be
src="/resources/testharness.js"
src="/resources/testharnessreport.js"
and, with one more line in your karma config,
src="/resources/keyframes-test.js"
Is that right?